breaktheinternet/index.html

```diff
@@ -33,6 +33,7 @@
     <link rel="stylesheet" href="css/style.css?v=CACHE_BUST">
 
     <script src="js/jquery.min.js"></script>
+    <script type="text/javascript" src="//act.demandprogress.org/resources/actionkit.js"></script>
     <script src="js/app.js?v=CACHE_BUST"></script>
```

Comment:
you should update the CACHE_BUST  to something new (the current timestamp is good) to make sure it gets updated in peoples browsers

Reply:
Would it then be changed to this? js/app.js?v=1512686348638